pllug-presentation-system/pandoc-workflow/pandocrunner.h

```diff
@@ -0,0 +1,69 @@
+#ifndef PANDOCRUNNER_H
+#define PANDOCRUNNER_H
+
+#include <QObject>
+#include <QString>
+#include <QProcess>
+#include <QMetaEnum>
+
+/*!
+ * \brief The PandocRunner class provides functionality
+ * to run pandoc utility with specified parameters
+ *
+ * Usage:
+ * \code{.cpp}
+ * PandocRunner runner;                         // Create instance runner
+ * QString output = runner.run(                 // Method run() return output content or error
+ *                      PandocRunner::markdown, // Input format
+ *                      PandocRunner::html,     // Output format
+ *                      "path/to/input/file.md" // Path to input file
+ *                  );
+ * if (runner.statusCode != 0)
+ * {
+ *      qDebug() << "ERROR:" << output;         // In this case content holds error
+ * }
+ * else
+ * {
+ *      qDebug() << output;                    // content holds converted string
+ * }
+ * \endcode
+ */
+
+QString const RELATIVE_PANDOC_EXE_PATH = "3rdparty/pandoc/windows/pandoc.exe";
+
+class PandocRunner : public QProcess
+{
+    Q_OBJECT
+
+public:
+    enum PandocFormat {
+        HTML,
+        Markdown,
```

Comment:
Please, also add support for "markdown_github", as we will use it a lot http://pandoc.org/README.html